bpf/lib/egress_gateway.h

```diff
@@ -129,6 +136,10 @@ bool egress_gw_snat_needed(__be32 saddr __maybe_unused,
 		return false;
 
 	*snat_addr = egress_gw_policy->egress_ip;
+#ifdef EGRESS_IFINDEX
```

Comment:
Who defines `EGRESS_IFINDEX`, and where is it defined?

Reply:
Hi @ysksuzuki! Once all the control plane bits are in place, this will be fed by the EGW manager into the `egress_gw_policy` entry. But that needs some additional work to create space in the struct, and migrate the existing map (for upgrades).